DESCRIPTION

```diff
@@ -79,7 +79,9 @@ Imports:
     crayon,
     rlang (>= 0.4.0),
     fastmap (>= 1.0.0),
-    withr
+    withr,
+    commonmark (>= 1.7),
+    glue (>= 1.3.1)
```

Comment:
I think this should be 1.3.2: https://github.com/tidyverse/glue/commit/2e3a4e01e088c3e9a702480e616905e5d1336495